app/presenters/variant_mailchimp_presenter.rb

```diff
@@ -13,9 +13,9 @@ def json
         {
           id: Digest::MD5.hexdigest("#{variant.sku}#{variant.id}"),
           title: variant.name || "",
-          sku: variant.sku,
-          price: variant.price.to_s,
-          url: "#{ENV['BASE_URL']}/#{variant.product.category&.permalink || 'products'}/#{variant.slug}",
+          sku: variant.sku || "",
```

Comment:
@chhga maybe it would be better to return the variant ID if SKU isn't available?